cli/src/main.rs

```diff
@@ -396,8 +400,9 @@ async fn run_test(test_args: TestArgs) -> anyhow::Result<i32> {
     match run_upload(
         upload_args,
         Some(command.join(" ")),
-        quarantine_run_result,
+        None, // don't re-run quarantine checks
         codeowners,
+        run_result.exec_start,
```

Comment:
nit: I think it's better to clone `run_result.exec_start` in this function—before `run_result` is passed to `run_quarantine`—rather than pass `run_result` as a reference and clone it... If I think about it more though, I feel like we should just pass `failures` and `exit_code` directly to `run_quarantine` rather than the entire `run_result`

Reply:
Done.